schunk_lwa4p_extended/config/arm_controller.yaml

```diff
@@ -11,7 +11,7 @@ joint_state_controller:
 
 ## joint trajectory controller
 joint_trajectory_controller:
-  type: position_controllers/JointTrajectoryController
+  type: effort_controllers/JointTrajectoryController
```

Comment:
why effort controller? Actually, it is not defined as EffortJointInterface..
https://github.com/ipa320/schunk_modular_robotics/blob/indigo_dev/schunk_description/urdf/lwa4p_extended/lwa4p_extended.transmission.xacro